contracts/BatchExchange.sol

```diff
@@ -438,7 +445,7 @@ contract BatchExchange is EpochTokenLocker {
                 usedAmount: 0
             })
         );
-        uint256 orderIndex = orders[msg.sender].length - 1;
+        uint16 orderIndex = (orders[msg.sender].length - 1).toUint16();
```

Comment:
Does this require a test?

Reply:
We used the toUint library in many places and only tested it in one spot. I think it is not necessary